pkg/common/client/config.go

```diff
@@ -265,9 +264,6 @@ func (b *configBuilder) Build(ctx context.Context) (*sarama.Config, error) {
 		config.ClientID = b.clientId
 	}
 
-	logger := logging.FromContext(ctx)
-	logger.Infof("Built Sarama config: %+v", config)
```

Comment:
Can't we have the message without the config?

Reply:
yeah, would be handy - and perhaps... in DEBUG ? :smile:

Reply:
Ok, then, and info message that the config is built, but the actual config in a debug message?

Reply:
even in DEBUG I don't think we want to print the user apikeys. What about printing the config before the apikey is set? Like maybe set it first to `XXX` and then substitute after printing the config? 

A better solution would be the override the String() function (haven't tried but should work)

Reply:
I can try override the String function